stdlib/Move.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+authors = ["MoveFuns DAO contributors"]
+name = "movefuns"
+version = "1.0.0"
+
+[addresses]
+std = "0x1"
+movefuns = "0x1"
```

Comment:
need to ensure the address.

Reply:
How to ensure the address instead of 0x1 without deploying on chain ? 
And can ensure that addresses from the same private_key are the same on different chain ?

Reply:
TODO 😅